protobuf/drand/api.proto

```diff
@@ -91,21 +91,17 @@ message PublicRandResponse {
 // PrivateRandRequest is the message to send when requesting a private random
 // value.
 message PrivateRandRequest {
-    // Request must contains a public key towards which to encrypt the private
-    // randomness.
-    ECIES request = 2;
+    // Request is the ECIES encryption of an ephemereal public key towards which
+    // to encrypt the private randomness. The format of the bytes is denoted by
+    // the ECIES encryption used by drand.
+    bytes request = 1;
```

Comment:
this is a breaking protobuf / protocol change. are we okay with that?

we should add a task for a test that alerts us whenever that happens

Reply:
I'm ok to do a breaking change for something that improves security :D 

But agreed with you, there should be some "cross version testing" or stg like that - not sure what's the best way to do that though.